### DISCUSSION TITLE
http://192.168.1.31:9999/ is unreachable? Only http://127.0.0.1:9999 works?

### DISCUSSION THREAD

**Discussion by dandaaa123123a** [xbapps/xbvr#1254] [Q&A]
How do I get my web url to work?
<img width="454" alt="Screenshot 2023-04-25 at 22 03 31" src="https://user-images.githubusercontent.com/131788135/234319691-c25ae88b-d1cb-4838-afe2-0bc8ccb7b1e7.png">

I need to access http://192.168.1.31:9999/ via my quest.

**Comment by dandaaa123123a** [comment]
<img width="556" alt="Screenshot 2023-04-25 at 22 04 08" src="https://user-images.githubusercontent.com/131788135/234319876-4addfbe9-b28c-4462-affc-703c4a44e271.png">

**Comment by dandaaa123123a** [comment]
Fixed

**Reply by browntable20** [reply]
what was the fix? having same issue
